packages/core/src/command/consolidated/__tests__/remove-contained-patients.test.ts

```diff
@@ -0,0 +1,216 @@
+import { Bundle, Resource, Patient, Composition } from "@medplum/fhirtypes";
+import { removeContainedPatients } from "../get-snapshot-local";
+
+describe("removeContainedPatients", () => {
+  const testPatientId = "test-patient-id";
```

Comment:
nit: if we can use uuidv7, it makes the test less prone to have to be updated in case we add a check that requires the IDs to be UUID. Non-blocking, though

Reply:
used faker.string.uuiid